website/content/en/status/report-2023-01-2023-03/clusteradm.adoc

```diff
@@ -12,22 +12,27 @@ In this quarter, the team has worked on the following:
 * Regular support for FreeBSD.org user accounts
 * Regular disk and parts support (and replacement) for all physical hosts and mirrors
 * Improve the PowerPC package builders +
++
 With new parts obtained through the FreeBSD Foundation, the builders now have new NVMEs with heatsinks and more memory.
 It helped seize the heat issue, and they are building packages faster now.
 * Decouple dynamic resources from the main websites +
++
 Work in coordination with doceng and webmaster to decouple dynamic resources from the websites, www.FreeBSD.org, and docs.FreeBSD.org.
 
 Work in progress:
 
 * Large-scale network upgrade at our primary site +
++
 New link:https://www.juniper.net/[Juniper] switches arrived at our primary site to replace the former ones.
 We thank link:https://www.juniper.net/[Juniper] for the donation.
 * Replace old servers in our primary site and a few mirrors +
++
 Besides the broken CI servers, we have a few old servers with broken disks and faulty PSUs.
 This task is in conjunction with the FreeBSD Foundation and donors/sponsors.
 * Deploy infrastructure to mirror the websites +
++
 Since the FreeBSD website is now mostly static, we have begun deploying infrastructure to mirror www.FreeBSD.org and docs.FreeBSD.org around the world in FreeBSD project-managed mirrors.
-* Create a new search database engine for internal FreeBSD.org searching needs like mailing list and docs
+* Create a new search database engine for internal FreeBSD.org searching needs like mailing list and docs.
 
 FreeBSD Official Mirrors Overview:
```

Comment:
With the colon here, should the subsequent paragraph become a list item? (Compare with uses of colons and list items above.)

Alternatively, use a subheading (without a colon).